--- docbook-utils-0.6.13/frontends/docbook.in.wiget	Sat Nov  2 18:45:10 2002
+++ docbook-utils-0.6.13/frontends/docbook.in	Mon Mar 17 20:16:20 2003
@@ -6,33 +6,7 @@
 # Give the location of the centralized catalog
 # First try the one with a version matching the version that is indicated in the document
   centralized-catalog)
-	if [ -n "$SGML_NORM" ]; then
-	  SGML_NORM_OPTS='-d'
-	else
-	  SGML_NORM=cat
-	  SGML_NORM_OPTS=
-	fi
-	SGML_VERSION=`"$SGML_NORM" $SGML_NORM_OPTS -- "$SGML_FILE" 2>/dev/null |
-			grep -i '<!DOCTYPE' |
-			head -n 1 |
-			sed 's/^.*DocBook\( XML\)\? V\([0-9][\.0-9]*\).*$/\2/'`
-	SGML_CATALOG=
-	for f in "${SGML_CATALOGS_DIR}/${SGML_XML}-docbook-${SGML_VERSION}"*".cat"
-	do 
-	  if [ -f "$f" ]
-	  then
-	    SGML_CATALOG=$f
-	    break
-	  fi
-	done
-	if [ -z "$SGML_CATALOG" ]
-	then
-	  if [ "${SGML_XML}" != "sgml" ]
-	  then SGML_CATALOG=${SGML_CATALOGS_DIR}/${SGML_XML}-docbook.cat
-	  else SGML_CATALOG=${SGML_CATALOGS_DIR}/catalog
-	  fi
-	fi
-	echo "$SGML_CATALOG"
+ 	echo "${SGML_CATALOGS_DIR}/catalog"
 	;;
 # Find a stylesheet on the model "html/docbook.dsl" or "print/docbook.dsl"
   style-sheet)
