--- docbook-utils-0.6.10/frontends/docbook.in.wiget	Mon Apr 15 18:58:21 2002
+++ docbook-utils-0.6.10/frontends/docbook.in	Sun May 26 02:03:10 2002
@@ -6,13 +6,7 @@
 # Give the location of the centralized catalog
 # First try the one with a version matching the version that is indicated in the document
   centralized-catalog)
-	SGML_VERSION=`grep -i '<!DOCTYPE' $SGML_FILE | head -n 1 | sed 's/\(V[0-9][\.0-9]*\).*$/\1/; s/^.* V\([0-9][\.0-9]*\)/\1/'`
-	if [ -f "${SGML_CATALOGS_DIR}/${SGML_XML}-docbook-${SGML_VERSION}"*".cat" ]
-	then echo "${SGML_CATALOGS_DIR}/${SGML_XML}-docbook-${SGML_VERSION}"*".cat"
-	elif [ "${SGML_XML}" != "sgml" ]
-	then echo "${SGML_CATALOGS_DIR}/${SGML_XML}-docbook.cat"
-	else echo "${SGML_CATALOGS_DIR}/catalog"
-	fi
+	echo "${SGML_CATALOGS_DIR}/catalog"
 	;;
 # Find a stylesheet on the model "html/docbook.dsl" or "print/docbook.dsl"
   style-sheet)
