--- docbook-utils-0.6/bin/docbook2dvi.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2dvi	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b dvi $*
+jw -f docbook -b dvi "$@"
--- docbook-utils-0.6/bin/docbook2html.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2html	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b html $*
+jw -f docbook -b html "$@"
--- docbook-utils-0.6/bin/docbook2man.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2man	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b man $*
+jw -f docbook -b man "$@"
--- docbook-utils-0.6/bin/docbook2pdf.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2pdf	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b pdf $*
+jw -f docbook -b pdf "$@"
--- docbook-utils-0.6/bin/docbook2ps.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2ps	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b ps $*
+jw -f docbook -b ps "$@"
--- docbook-utils-0.6/bin/docbook2rtf.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2rtf	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b rtf $*
+jw -f docbook -b rtf "$@"
--- docbook-utils-0.6/bin/docbook2tex.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2tex	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b tex $*
+jw -f docbook -b tex "$@"
--- docbook-utils-0.6/bin/docbook2texi.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2texi	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b texi $*
+jw -f docbook -b texi "$@"
--- docbook-utils-0.6/bin/docbook2txt.@	Thu Nov  9 22:07:24 2000
+++ docbook-utils-0.6/bin/docbook2txt	Mon Mar 12 17:26:33 2001
@@ -1,2 +1,2 @@
 #! /bin/sh
-jw -f docbook -b txt $*
+jw -f docbook -b txt "$@"
